app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -42,10 +42,19 @@ def destroy
         end
       end
 
+      def get_moocs
+        mooc_school = CodeSchool.where(is_mooc: "true")
```

Comment:
A couple things here.

1 - Small nit, can we change the variable from singular to plural `mooc_schools`
2 - `is_mooc` is a `boolean` type, not string. So this would actually be `CodeSchool.where(is_mooc: true)`